config/peerpods/podvm/aws-podvm-image-cm.yaml

```diff
@@ -33,3 +33,8 @@ data:
   # NVIDIA GPU vars
   NVIDIA_DRIVER_VERSION: "535"
   NVIDIA_USERSPACE_VERSION: "1.13.5-1"
+
+  # Pause image vars
+  PAUSE_IMAGE_REPO: "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256"
+  PAUSE_IMAGE_VERSION: "ac97982e845413b2b7377171d1b4c85d2b2cfda174e8251e3049d4843cea40e1"
```

Comment:
would this be different for other architectures?

Reply:
I think so

Reply:
The OCP pause image is multi-arch.